### ISSUE TITLE
Tests failing with Python 3.8

### ISSUE THREAD

**Issue by MageJohn** [wren-lang/wren#742]
I'm on Manjaro Linux, and the tests are failing for me due to issues with Python trying to parse invalid utf-8 being used as part of a test. In Manjaro `/usr/bin/python` is Python 3 by default (specifically Python 3.8 currently), not Python 2, and I confirmed that the tests don't fail when I used `python2.7 utils/test.py` instead. 

The output from running with Python 3:
```
(bin/wrend) Passed: 0 Failed: 0 Skipped: 0 Traceback (most recent call last):
  File "util/test.py", line 383, in <module>
    walk(join(WREN_DIR, 'test'), run_test, ignored=['api', 'benchmark'])
  File "util/test.py", line 304, in walk
    walk(nfile, callback)
  File "util/test.py", line 306, in walk
    callback(nfile)
  File "util/test.py", line 365, in run_test
    run_script(WREN_APP, path, "test")
  File "util/test.py", line 346, in run_script
    if not test.parse():
  File "util/test.py", line 67, in parse
    for line in file:
  File "/usr/lib/python3.8/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xab in position 65: invalid start byte
```

With a little print() debugging, I found the test causing the error, namely [test/regression/442-000088.wren](https://github.com/wren-lang/wren/blob/c7087652ee34ff9d2d1fd697bd8a4eac22a96182/test/regression/442-000088.wren)

**Comment by underscorediscovery** [comment]
The tests have been fixed in the 0.3.0 branch and when that is released, it'll be resolved. Thanks for the report.

**Comment by MageJohn** [comment]
I should probably have waited a few minutes then before making a patch. 😂
